tests/foreman/cli/test_user.py

```diff
@@ -1091,6 +1093,45 @@ def test_positive_remove_role(self):
                 user = User.info({'id': user['id']})
                 self.assertNotIn(role_name, user['roles'])
 
+    @pytest.mark.skip_if_open("BZ:1763816")
+    @tier2
+    def test_positive_last_login_for_new_user(self):
+        """Create new user with admin role and check last login updated for that user
+
+        :id: 967282d3-92d0-42ce-9ef3-e542d2883408
+
+        :expectedresults: last login should be updated for user after login using hammer
+
+        :BZ: 1765052
```

Comment:
BZ:1763816

Reply:
@ntkathole it was link this bug, comment https://github.com/SatelliteQE/robottelo/pull/7467#issuecomment-553347623

Reply:
No, really, you changed the bug ID in `pytest.mark.skip` but not here, I think you should change this to 1763816, too.

Reply:
yeah too much of brain usage, got you